draft-ietf-mls-architecture.md

```diff
@@ -642,21 +671,21 @@ members. No assumptions are made about the format of the payload.
 The protocol aims to be compatible with federated environments. While
 this document does not specify all necessary mechanisms required for
 federation, multiple MLS implementations can interoperate to form
-federated systems if they use compatible authentication mechanisms
-and infrastructure functionalities.
+federated systems if they use compatible authentication mechanisms, 
+ciphersuites, and infrastructure functionalities.
 
-### Compatibility with future versions of MLS
+### Compatibility with Future Versions of MLS
 
 It is important that multiple versions of MLS be able to coexist in
-the future.  Thus, MLS offers a version negotiation mechanism; this
+the future. Thus, MLS offers a version negotiation mechanism; this
 mechanism prevents version downgrade attacks where an attacker would
 actively rewrite messages with a lower protocol version than the ones
 originally offered by the endpoints. When multiple versions of MLS are
 available, the negotiation protocol guarantees that the version agreed
 upon will be the highest version supported in common by the group.
 
 In MLS 1.0, the creator of the group is responsible for selecting the
-best ciphersuite proposed across clients. Each client is able to
+best ciphersuite supported across clients. Each client is able to
```

Comment:
I think “offered” should be used here. The clients are not forced to offer everything they support.